tomviz/CentralWidget.cxx

```diff
@@ -53,6 +53,10 @@
 # include "dax/ModuleStreamingContour.h"
 #endif
 
+Q_DECLARE_METATYPE(vtkSmartPointer<vtkImageData>)
```

Comment:
This was the issue with crashing when the application was shut down too quickly.  The background thread needs to own a reference to the vtk object, otherwise the VTK object can be deleted while the background object is still reading data from it to create the histogram.  I figured it was safer to pass the smart pointer to avoid potential race conditions.